### PR TITLE
Fix #2600: Turn on docker build

### LINKED ISSUE #2600: Restore docker functionality
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**

Two outcomes here:
* `docker-compose up` starts the app again, running processes for jsbundling and cssbundling. This should be reflected in the current docker-compose.yml file
* the `dockercheck` CI task runs and succeeds again

* **What feature or behavior is this required for?**

cleaning up the mess left by removing webpacker; builds

* **How could we solve this issue? (Not knowing is okay!)**

no idea. good luck @elimbaum !

* **Anything else?**
see #2594

### PR DESCRIPTION
**Pull request by xmunoz**
I have removed the css and js conatiners. Here is the error they were throwing:
```
Usage Error: Couldn't find the node_modules state file - running an install might help (findPackageLocation)
```

To re-add those containers: #2606

### CODE REVIEW

**Comment by xmunoz** [comment]
Still getting some errors:
```
jsbuild_1   | ✘ [ERROR] Could not resolve "jquery-ujs"
jsbuild_1   | 
jsbuild_1   |     app/javascript/application.js:4:15:
jsbuild_1   |       4 │ import {} from 'jquery-ujs'
jsbuild_1   |         ╵                ~~~~~~~~~~~~
jsbuild_1   | 
jsbuild_1   |   You can mark the path "jquery-ujs" as external to exclude it from the bundle, which will remove this error.
jsbuild_1   | 
jsbuild_1   | ✘ [ERROR] Could not resolve "bootstrap"
jsbuild_1   | 
jsbuild_1   |     app/javascript/application.js:7:27:
jsbuild_1   |       7 │ import * as bootstrap from "bootstrap";
jsbuild_1   |         ╵                            ~~~~~~~~~~~
jsbuild_1   | 
jsbuild_1   |   You can mark the path "bootstrap" as external to exclude it from the bundle, which will remove this error.
jsbuild_1   | 
jsbuild_1   | ✘ [ERROR] Could not resolve "jquery"
jsbuild_1   | 
jsbuild_1   |     app/javascript/src/jquery.js:1:19:
jsbuild_1   |       1 │ import jquery from 'jquery';
jsbuild_1   |         │                    ~~~~~~~~
jsbuild_1   |         ╵                    "./jquery"
jsbuild_1   | 
jsbuild_1   |   Use the relative path "./jquery" to reference the file "app/javascript/src/jquery.js". Without the leading "./", the path "jquery" is being interpreted as a package path instead.
jsbuild_1   | 
jsbuild_1   | ✘ [ERROR] Could not resolve "@fortawesome/fontawesome-free/js/all"
jsbuild_1   | 
jsbuild_1   |     app/javascript/src/fontawesome.js:2:7:
jsbuild_1   |       2 │ import '@fortawesome/fontawesome-free/js/all';
jsbuild_1   |         ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
jsbuild_1   | 
jsbuild_1   |   You can mark the path "@fortawesome/fontawesome-free/js/all" as external to exclude it from the bundle, which will remove this error.
jsbuild_1   | 
jsbuild_1   | ✘ [ERROR] Could not resolve "turbolinks"
jsbuild_1   | 
jsbuild_1   |     app/javascript/src/turbolinks.js:3:23:
jsbuild_1   |       3 │ import Turbolinks from 'turbolinks';
jsbuild_1   |         │                        ~~~~~~~~~~~~
jsbuild_1   |         ╵                        "./turbolinks"
jsbuild_1   | 
jsbuild_1   |   Use the relative path "./turbolinks" to reference the file "app/javascript/src/turbolinks.js". Without the leading "./", the path "turbolinks" is being interpreted as a package path instead.
jsbuild_1   | 
jsbuild_1   | ✘ [ERROR] Could not resolve "stupid-table-plugin"
jsbuild_1   | 
jsbuild_1   |     app/javascript/src/table_sorting.js:2:7:
jsbuild_1   |       2 │ import 'stupid-table-plugin';
jsbuild_1   |         ╵        ~~~~~~~~~~~~~~~~~~~~~
jsbuild_1   | 
jsbuild_1   |   You can mark the path "stupid-table-plugin" as external to exclude it from the bundle, which will remove this error.
jsbuild_1   | 
jsbuild_1   | 6 errors
jsbuild_1   | [watch] build finished, watching for changes...
cssbuild_1  | Error: Can't find stylesheet to import.
cssbuild_1  |   ╷
cssbuild_1  | 1 │ @import 'bootstrap/scss/bootstrap';
cssbuild_1  |   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^
cssbuild_1  |   ╵
cssbuild_1  |   app/javascript/styles/bootstrap.scss 1:9               @import
cssbuild_1  |   app/assets/stylesheets/application.bootstrap.scss 2:9  root stylesheet
cssbuild_1  | 
cssbuild_1  | Sass is watching for changes. Press Ctrl-C to stop.
cssbuild_1  | 

```

File: .docker/Dockerfile [hunk 1]
```diff
@@ -27,12 +27,13 @@ RUN apt-get update && apt-get upgrade -y && \
...
-    apt-get install nodejs -y
+RUN curl -sL https://deb.nodesource.com/setup_16.x | sudo bash -
+RUN apt-get install nodejs -y
 
-RUN npm install -g yarn
-RUN yarn set version berry
-RUN yarn set version 3.2.0
+# set up yarn
+RUN corepack enable
+RUN yarn set version 3.2.1
```
Comment (xmunoz) [inline .docker/Dockerfile:35]: PR #2581 missed a spot

File: .docker/Dockerfile [hunk 2]
```diff
@@ -62,4 +63,8 @@ RUN ahab_dep=${AHAB_DEPENDENCIES} && \
 # Move the rest of the app over
 COPY . ${DARIA_DIR}
 
+# Build the js and css
+RUN yarn build
```
Comment (colinxfleming) [inline .docker/Dockerfile:67]: the rub here is that these commands do a similar-to-webpack thing, where they watch for changes in development. but restoring the build definitely more valuable than getting this perfect, so ⏺️

**Review by colinxfleming — APPROVED** [review]
fire when ready, we'll fix the watch thing later

**Pull request merged**